### PR TITLE
CR-1107824: Align xocl_debug log with profiling data

### PR DESCRIPTION
**Pull request by jvillarre**
This pull request changes the internal OpenCL debug file generation (enabled with xocl_debug=true) to use the zero point inside time_ns() instead of its own local zero point.  Removing this local zero time aligns the timestamps in the generated debug file with any timestamps generated by profiling.

### CODE REVIEW

File: src/runtime_src/xocl/core/debug.cpp [hunk 1]
```diff
@@ -42,7 +42,6 @@ namespace {
 
 static bool s_debug_on = false;
 static std::string s_debug_log;
-static unsigned long s_zero = xocl::time_ns();
```
Comment (stsoe) [inline src/runtime_src/xocl/core/debug.cpp:45]: Let's just do 
`s_zero = get_time_zero(); `

and get_time_zero() returns 0 by default but can be changed to original behavior under ini key if for some reason the old behavior is needed.

File: src/runtime_src/xocl/core/debug.cpp [hunk 2]
```diff
@@ -68,10 +67,10 @@ struct info
...
         m_times[CL_RUNNING] = m_times[CL_COMPLETE];
     ostr << id << " " << m_command_type << " "
-         << (m_times[CL_QUEUED]-s_zero) << " "
-         << (m_times[CL_SUBMITTED]-s_zero) << " "
-         << (m_times[CL_RUNNING]-s_zero) << " "
-         << (m_times[CL_COMPLETE]-s_zero);
+         << m_times[CL_QUEUED] << " "
+         << m_times[CL_SUBMITTED] << " "
+         << m_times[CL_RUNNING] << " "
+         << m_times[CL_COMPLETE];
```
Comment (stsoe) [inline src/runtime_src/xocl/core/debug.cpp:73]: No change per above.

File: src/runtime_src/xocl/core/debug.cpp [hunk 3]
```diff
@@ -187,9 +186,6 @@ init()
 
   ::event::init();
 
-  // reset time zero
-  s_zero = xocl::time_ns();
```
Comment (stsoe) [inline src/runtime_src/xocl/core/debug.cpp:191]: get_time_zero()

**Review by stsoe — CHANGES_REQUESTED** [review]
Let's keep the existing code for easy resurrection in case it is needed.

File: src/runtime_src/xocl/core/debug.cpp (hunk 1 above)
Comment (jvillarre) [reply src/runtime_src/xocl/core/debug.cpp:45]: Added get_time_zero function as requested.

File: src/runtime_src/xocl/core/debug.cpp (hunk 2 above)
Comment (jvillarre) [reply src/runtime_src/xocl/core/debug.cpp]: Reverted change

File: src/runtime_src/xocl/core/debug.cpp (hunk 3 above)
Comment (jvillarre) [reply src/runtime_src/xocl/core/debug.cpp:191]: Replaced with call to get_time_zero

**Comment by dayeh-xilinx** [comment]
retest this please.

**Review by stsoe — APPROVED** [review]

**Pull request merged**